app/src/ui/branches/branches-container.tsx

```diff
@@ -45,6 +45,10 @@ interface IBranchesContainerProps {
   readonly isLoadingPullRequests: boolean
 
   readonly currentBranchProtected: boolean
+
+  readonly selectedUncommittedChangesStrategy: UncommittedChangesStrategy
+
+  readonly hasChangesAndStash: boolean
```

Comment:
non-blocking idea for a maybe more legible name:
```suggestion
  readonly couldOverwriteStash: boolean
```